website/docs/tutorials/react.md

```diff
@@ -633,7 +672,7 @@ The last message in our component is again a bit specific:
 `lastLogin` is a date object, and we need to format it properly. Dates are formatted differently in different languages, but we don't have to do this manually. The heavy lifting is done by the [Intl object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl), we'll just use [`i18n.date()`](/docs/ref/core.md#i18n.date) function. The `i18n` object can be accessed by [`useLingui`](/docs/ref/react.md#uselingui) hook:
 
 ```jsx title="src/Inbox.js"
-import { useLingui } from "@lingui/react";
+import { useLingui } from "@lingui/macro";
```

Comment:
the sentence above also need to be updated, it still describes the react hook

Reply:
i think we can revert here import from react, since it doesn't matter in this case what flavor to use.